debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/PostgresConnectorIT.java

```diff
@@ -520,6 +520,273 @@ public void shouldReceiveChangesForChangePKColumnDefinition() throws Exception {
         }
     }
 
+    @Test
+    public void shouldReceiveChangesForChangeColumnDefault() throws Exception {
```

Comment:
@gunnarmorling This is the test you requested, to show how the connector responds when the default value changes